common/ts/sonarqube/__tests__/TaskReport-test.ts

```diff
@@ -23,3 +28,57 @@ serverUrl=http://sonar`,
 
   tmpReport.removeCallback();
 });
+
+
+it('should parse all reports', async () => {
```

Comment:
Miscellaneous: these tests check the pre-existing behaviour i.e. not affected the fix. I added these tests mainly to build familiarity with the test and mocking frameworks.